.gitignore

```diff
@@ -6,5 +6,3 @@
 # Files generated/used by direnv
 .envrc
 .direnv/
-
-Cargo.lock
```

Comment:
I've had multiple build issues on both my WS because it was missing. Please read [updated guidelines post](https://blog.rust-lang.org/2023/08/29/committing-lockfiles.html) for more details.

Reply:
TIL!
Thanks!